pkgs/development/python-modules/crytic-compile/default.nix

```
@@ -0,0 +1,22 @@
+{ stdenv, buildPythonPackage, fetchPypi, python3Packages }:
```

for python-modules, you import individual packages
```suggestion
{ stdenv, buildPythonPackage, fetchPypi, pysha3 }:
```

---

Done